### PR TITLE
[Merged by Bors] - Let the project page support GitHub's new ability to display open source licenses

### PR DESCRIPTION
**Pull request by C-BJ**
Change _LICENSE-APACHE_ and _LICENSE-MIT_ file location
Delete _LICENSE_
You can make the license in about on bevy's GitHub page display as **Apache-2.0, MIT licenses found** instead of **View license**

### CODE REVIEW

**Review by alice-i-cecile — COMMENTED** [review]
Is there a way to preview what GitHub does with this?

**Comment by C-BJ** [comment]
> Is there a way to preview what GitHub does with this?

https://github.com/C-BJ/bevy

**Review by alice-i-cecile — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Additionally, we would like to thank the [Amethyst](https://github.com/amethyst/
 Bevy is free and open source! All code in this repository is dual-licensed under either:
 
 * MIT License ([LICENSE-MIT](LICENSE-MIT) or [http://opensource.org/licenses/MIT](http://opensource.org/licenses/MIT))
-* Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0))
+* Apache License Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0))
```
Comment (bjorn3) [inline README.md:104]: The official name is "Apache License, Version 2.0", not "Apache License Version 2.0"

Comment (C-BJ) [reply README.md:104]: > The official name is "Apache License, Version 2.0", not "Apache License Version 2.0"

oh, my bad

**Comment by alice-i-cecile** [comment]
Yep, CI is being flaky... I'm going to wait a few hours and then retry merging.

Anyone who's a member of the Bevy org can attempt to merge this again if I forget; just use `bors retry`.

**Comment by alice-i-cecile** [comment]
bors retry

**Comment by DJMcNab** [comment]
I think this really should have had @cart's sign off? 
I think any PR which does licensing things probably should (as well as any PR which changes the set of files at the root, probably)

I don't think this materially changes anything in terms of licensing - the documents are only included for reference and the actual application is [the section in the README](https://github.com/bevyengine/bevy/blob/main/README.md#license).
This is important to think about because we do have some things not under the MIT or Apache 2.0 licenses, most notably under `assets`, but in a handful of other places too.

**Comment by alice-i-cecile** [comment]
That's fair. We should add those criteria to the list of controversial items. I'll make a PR right now.

**Comment by cart** [comment]
Yup I would have appreciated being consulted here (and would like to be for similar changes in the future).

**Comment by cart** [comment]
That being said I do agree with this change.

**Pull request closed without merging**